guide/src/getting_started.md

```diff
@@ -95,6 +95,13 @@ Make sure that the rust you want to be able to access from Python is compiled in
 
 
 ```toml
+[package]
+# Name of the package. If you already have a package defined in `Cargo.toml`, you can remove
```

Comment:
I was a bit confused by "name of the package", which refers only to the first entry, while the rest of the comment refers to the whole section.

Reply:
thanks! addressed in ca1c9c1d - changed to

```
# If you already have [package] information in `Cargo.toml`, you can ignore
# this section!
[package]
# `name` here is name of the package.
name = "pyo3_start"
# these are good defaults:
version = "0.1.0"
edition = "2021"
```